trunk/NDHMS/Routing/module_NWM_io.F

```diff
@@ -839,8 +839,8 @@ subroutine output_chrt_NWM(domainId)
             ! Extract valid range into a 1D array for placement. 
             varRange(1) = fileMeta%validMinComp(iTmp)
             varRange(2) = fileMeta%validMaxComp(iTmp)
-            varRangeReal(1) = fileMeta%validMinReal(iTmp)
-            varRangeReal(2) = fileMeta%validMaxReal(iTmp)
+            varRangeReal(1) = real(fileMeta%validMinReal(iTmp))
+            varRangeReal(2) = real(fileMeta%validMaxReal(iTmp))
```

Comment:
It strikes me that we should rename "validMaxReal" to "validMaxDouble". but I'm content to do that later.